crates/astria-sequencer/src/app/mod.rs

```diff
@@ -468,16 +584,20 @@ impl App {
 
                 return Ok(());
             }
-            self.metrics.increment_process_proposal_skipped_proposal();
-            debug!(
-                "our validator address was set but we're not the proposer, so our previous \
-                 proposal was skipped, executing block"
-            );
-            self.executed_proposal_fingerprint = None;
+            ProposalFingerprintStatus::Unset => {}
```

Comment:
this case should also call `update_state_for_new_round` - move `update_state_for_new_round` after the match statement